test/foundry/Regular_Tests/Fuzzing.t.sol

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.10;
+
+import "./Setup.sol";
+
+/**
+ * @title JetStakingV1 foundry fuzzing tests
+ * @author Lance Henderson
+ *
+ * @dev Fuzzing test suite for JetStakingV1 written in solidity using Foundry
+ * 
+ *      FuzzingTests implements a testing technique known  as "fuzzing". This technique 
+ *      involves testing a specific scenario with a wide array of test values, with the 
+ *      goal of finding values which create unexepcted behaviour in the system.
+ *    
+ *      A detailed description of  what each test does is provided in  the space  above 
+ *      the test itself.
+ */
+
+contract FuzzingTests is Setup {
+
+    /*/////////////////////////////////////////////////////////////
+                    STAKE / UNSTAKE (PARAM: AMOUNT)
+    /////////////////////////////////////////////////////////////*/
+
+    /**
+     * @dev Stake and immediately unstake from the jetStaking contract. No reward 
+     *      tokens nor aurora tokens should accrue. This is checked in the assert.
+     *
+     * @param x Amount of aurora tokens to stake
+     */
+    function testStakeAndWithdraw(uint256 x) public {
```

Comment:
If it is in the same block, so it is better to rename the function `testStakeAndWithdrawInTheSameBlock`.